app/controllers/lines_controller.rb

```diff
@@ -1,6 +1,11 @@
 # Single-serving controller for setting current line for a user.
 class LinesController < ApplicationController
-  def new; end
+  def new
+    if LINES.count == 1
```

Comment:
I'd like a controller test for new to go with this behavior too, but I'm happy to handle that since it's a little pedantic.

This is pretty much dead on, by the way 💯